utils/instructionTools.ts

```diff
@@ -227,6 +233,7 @@ export async function getSolTransferInstruction({
       form.amount!,
       governedTokenAccount.extensions.mint.account.decimals
     )
+    console.log(governedTokenAccount.extensions.mint.account.decimals)
```

Comment:
nit: remove console log

Reply:
Sorry for late reply, I removed the console.log